<!-- launch all dpl nodes -->
<launch>
  <group ns="dpl">
<<<<<<< HEAD
=======
    <!-- robot's id in ROS ecosystem -->
    <param name="robot_id" type="int" value="0" />

    <!-- parameters of wheel odometry setup -->
    <param name="frameIDodom" value="/wheel_odom"/>
    <param name="odomPublTopic" value="wheel_odom"/>

>>>>>>> parametrization of setup for motor odometry broadcaster
    <!-- hal is foreign dpl member -->
    <include file="$(find p3dx_hal_vrep)/launch/hal_vrep.launch" />

    <!-- dpl members -->
    <node pkg="p3dx_dpl" type="motor_controler" name="motor_ctl" respawn="true" />
    <node pkg="p3dx_dpl" type="imu_provider" name="imu_provider" respawn="true" />
    <node pkg="p3dx_dpl" type="odometry_provider" name="odom_provider" respawn="true" />
    <node pkg="p3dx_dpl" type="odom_tf_broadcaster" name="odom_tf" respawn="true" />
  </group>
</launch>